fragments/shared/validation/success_on_first_boot.ks

```diff
@@ -0,0 +1,59 @@
+# Report success if no errors have been reported on the first boot.
+
+%post
+
+# Create a systemd service.
+cat > /etc/systemd/system/kickstart-test.service << EOF
+
+[Unit]
+Description=The kickstart test service
+After=initial-setup.service
+After=multi-user.target
+After=graphical.target
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh /usr/libexec/kickstart-service.sh
+
+[Install]
+WantedBy=graphical.target
+WantedBy=multi-user.target
+
+EOF
+
+# Create a script with the actual test. Print errors to stdout.
+# IMPORTANT: This file should be rewritten in tests!
+touch /usr/libexec/kickstart-test.sh
+
+# Create a script for the service
+cat > /usr/libexec/kickstart-service.sh << 'EOF'
```

Comment:
I wonder why you have `'` around the `EOF` here.

Reply:
Please, see the comment https://github.com/rhinstaller/kickstart-tests/pull/464#discussion_r558914963.